civitai-downloader-v2/src/api/client.py

```
@@ -8,6 +8,8 @@
 import asyncio
 from typing import Dict, Any, Optional, AsyncIterator, List
 import sys
+import math
```

Ruff (0.12.2)

`math` imported but unused

Remove unused import: `math`

(F401)